Userland/Applications/PixelPaint/Layer.cpp

```diff
@@ -90,6 +90,18 @@ void Layer::set_name(String name)
     m_image.layer_did_modify_properties({}, *this);
 }
 
+void Layer::flip(Gfx::Orientation orientation)
+{
+    m_bitmap = m_bitmap->flipped(orientation).release_nonnull();
```

Comment:
It would be cool to flip in-place here, to avoid creating a whole new bitmap.
As it stands now, we simply don't handle the case where `Bitmap::flipped()` fails and returns `nullptr`.

Reply:
I made the requested changes for layer flipping, it should be good now.